osu.Game/Online/Rooms/MultiplayerPlaylistItem.cs

```diff
@@ -53,22 +53,12 @@ public class MultiplayerPlaylistItem
         [Key(9)]
         public DateTimeOffset? PlayedAt { get; set; }
 
-        public MultiplayerPlaylistItem()
-        {
-        }
+        [Key(10)]
+        public double StarRating { get; set; }
 
-        public MultiplayerPlaylistItem(PlaylistItem item)
+        [SerializationConstructor]
+        public MultiplayerPlaylistItem()
```

Comment:
I'm 99% sure we don't need this default `ctor`, but leaving it here because I don't want to accidentally break everything server comms wise.